src/test/java/org/takes/facets/auth/social/PsLinkedinTest.java

```diff
@@ -24,32 +24,214 @@
 
 package org.takes.facets.auth.social;
 
+import java.io.FileInputStream;
 import java.io.IOException;
-import org.junit.Ignore;
+import java.net.HttpURLConnection;
+import javax.json.Json;
+import nl.jqno.equalsverifier.EqualsVerifier;
+import nl.jqno.equalsverifier.Warning;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.takes.Response;
+import org.takes.facets.auth.Identity;
+import org.takes.misc.Href;
+import org.takes.rq.RqFake;
+import org.takes.rs.RsWithBody;
+import org.takes.rs.RsWithStatus;
+import org.takes.rs.RsWithType;
 
 /**
  * Test case for {@link PsLinkedin}.
  * @author Dmitry Zaytsev (dmitry.zaytsev@gmail.com)
  * @version $Id$
  * @since 0.11.3
+ * @checkstyle MultipleStringLiteralsCheck (500 lines)
  */
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
@ikhvostenkov this should also be removed (please no `@Mock` annotations in tests)